locales/en/breaches.ftl

```diff
@@ -15,6 +15,11 @@ emails-monitored =
      *[other] { $count } of { $total } emails monitored
   }
 
+## Breaches resolved filter
+
+label-unresolved = Unresolved breaches
```

Comment:
Can we use more semantic IDs, also to discourage reuse in other contexts? e.g. ´filter-label-unresolved`.

Reply:
Since @toufali is out on PTO, I've addressed this